src/sqs-instance.js

```diff
@@ -1,3 +1,5 @@
 import AWS from 'aws-sdk';
+import config from 'config';
 
-export default new AWS.SQS({ apiVersion: '2012-11-05' });
+const api_version = config.get('aws.api_version');
+export default new AWS.SQS({ api_version: api_version });
```

Comment:
Very unsure - is this the correct way of referencing the value, or can one just insert the const without brackets or key? 

i.e. …AWS.SQS(api_version);